src/bin-utils/index.js

```diff
@@ -161,22 +171,41 @@ function requireDefaultFromModule(modulePath) {
   }
 }
 
+function stringifyScript({name, description, script}) {
+  const coloredName = chalk.green(name)
+  const coloredScript = chalk.gray(script)
+  let line
+  if (description) {
+    line = [coloredName, chalk.white(description), coloredScript]
+  } else {
+    line = [coloredName, coloredScript]
+  }
+  return line.join(' - ').trim()
+}
+
+function specificHelpScript({scripts}, scriptName) {
+  const availableScripts = getAvailableScripts(scripts)
+  const script = find(
```

Comment:
How does prefix-matches help and work ? Also, I am not sure I fully understand what `prefix` means in this context. Can you explain ?